rlinf/runners/reasoning_runner.py

```diff
@@ -81,8 +81,14 @@ def __init__(
         self.actor_channel = Channel.create("Actor", local=True)
 
         # Configurations
-        self.compute_ref_logprobs = self.cfg.algorithm.kl_beta > 0
-        self.recompute_logprobs = self.cfg.algorithm.recompute_logprobs
+        self.compute_ref_logprobs = (
+            self.cfg.algorithm.kl_beta > 0
+            or self.cfg.algorithm.get("reinpp_kl_beta", 0) > 0
+        )
+        self.recompute_logprobs = (
```

Comment:
Same here. Move this to config rather than checking this everywhere throughout runner, rollout, and training.